db/migrations/0003_alter_ticket_unique_together.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 4.0.2 on 2023-11-30 21:15
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0002_user_order_ticket_alter_movie_actors_and_more'),
+    ]
+
+    operations = [
+        migrations.AlterUniqueTogether(
+            name='ticket',
+            unique_together={('movie_session', 'row', 'seat')},
+        ),
+    ]
```

Comment:
It is a good practice for Django projects to include one migration per app per pull request to not overload the `migrations` folder in the future. Add one migration to your pull request.

Reply:
It will not work otherwise, already tried, read at the top, before the last commit